request/builder.js

```diff
@@ -328,20 +345,25 @@ class ContractsListingRequestBuilder {
     const contractIdEncoded = new TextEncoder('utf-8').encode(this.contractId);
 
     const buffer = new Uint8Array(
-        contractIdEncoded.byteLength + certHolderId.byteLength +
-        certVersion.byteLength);
+        contractIdEncoded.byteLength + certVersion.byteLength
+        + certHolderId.byteLength);
     let offset = 0;
 
-    buffer.set(contractIdEncoded, offset);
-    offset += contractIdEncoded.byteLength;
     buffer.set(certHolderId, offset);
     offset += certHolderId.byteLength;
     buffer.set(certVersion, offset);
+    offset += certVersion.byteLength;
+    buffer.set(contractIdEncoded, offset);
```

Comment:
It isn't OK to change the order, is it ?

Reply:
After i checked the protofile, the latest version is the correct one, we made a mistake in the past...here is the proto file
```
message ContractsListingRequest {
    string cert_holder_id = 1;
    uint32 cert_version = 2;
    string contract_id = 3;
    bytes signature = 4;
}
```